Pre-emptive fix for portrait images and grid

ChrisBAshton opened the pull request:
Pre-empts fixing an issue with portrait images (see comments on the PR).

Not intended to be merged into #1501 before that gets merged into `latest`.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

dr3 commented on .env:
```diff
@@ -1,4 +1,6 @@
-# To ignore and keep this file we have a command in `postshrinkwrap` which assumes this file is not changing
-# The command is `git update-index --assume-unchanged .env` 
-# Source: https://wildlyinaccurate.com/git-ignore-changes-in-already-tracked-files/
+SIMORGH_BASE_URL=http://localhost:7080
```
This file needs resetting

ChrisBAshton replied:
Why does this keep happening?! 😫

dr3 reviewed (changes requested):
Can you also update `layoutGridItemSmall` as I think that has the same issue

ChrisBAshton commented:
> Can you also update layoutGridItemSmall as I think that has the same issue

I don't think it applies there? `layoutGridItemSmall` isn't used on paragraphs (it's only used for images where `height > width` AFAIK) and this is a fix for adding margin to paragraphs on mobile screens.

NB this is what `layoutGridItemSmall` currently looks like (ignore the image, I had to change the fixture data image dimensions but didn't bother changing the src)

![Screen Shot 2019-03-26 at 08 42 08](https://user-images.githubusercontent.com/5111927/54982607-18738c00-4fa3-11e9-847d-372deb491e0b.png)

ChrisBAshton committed:
fixes layoutGridItemSmall

had to change the 'max-width' parts as otherwise there'd be an exact pixel size screen that was in between media query sizes and would get the default (mobile) styling

ChrisBAshton commented:
Talked IRL, understood that we probably don't want the 'spacing' on portrait images and should adjust the class accordingly. Have done so here:

https://github.com/bbc/simorgh/pull/1490/commits/22fc2ed9e8aeb41e4d5de91a0b8d27b1020ab1ad

Video:

![ezgif-2-3114ce94cb0a](https://user-images.githubusercontent.com/5111927/54993420-a0fd2700-4fb9-11e9-8534-c836ee36819b.gif)

dr3 commented on src/app/lib/layoutGrid.js:
```diff
@@ -135,18 +134,17 @@ export const layoutGridItemMedium = css`
-    grid-column: 1 / span 6;
+  grid-column: 1 / span 6;
+
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
```
This doesnt match zeplin. At 768px zeplin spans 4 columns but this spans 5

ChrisBAshton replied:
Let's look at it with UX tomorrow.

ChrisBAshton commented:
Have raised #1501 - let's get that across the board.

We can look at this PR with UX later but it's not a priority for launch, unlike #1501 is.
